.gitignore

```diff
@@ -95,7 +96,3 @@ coverage/
 !**/ios/**/default.mode1v3
 !**/ios/**/default.mode2v3
 !**/ios/**/default.pbxuser
-!**/ios/**/default.perspectivev3
-!/packages/flutter_tools/test/data/dart_dependencies_test/**/.packages
-
-android/app/google-services.json
```

Comment:
Sorry, just delete this line 'android/app/google-services.json' @@!
And add .env